cloud/pulsar-admin/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+
+The `pulsar-admin` is a CLI tool written by Java language for the Apache Pulsar project.
+
+# Prerequisites
+
+- Pulsar broker 2.7.0-742fc5c9b+
+
+> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When the 2.6.1 release is complete, you can also use the official 2.6.1 version.
```

Comment:
```suggestion
> You can get this tarball from [binary](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
```

Reply:
`bintray` is the name of a website, not typo.